compute_endpoint/globus_compute_endpoint/endpoint/default_config.py

```diff
@@ -3,6 +3,7 @@
 from parsl.providers import LocalProvider
 
 config = Config(
+    display_name=None,  # If None, defaults to the endpoint name
     executors=[
```

Comment:
Suggest to let the default be the default (which is set to `None` as of this PR), and don't put this in.